odl/solvers/advanced/chambolle_pock.py

```diff
@@ -108,6 +105,39 @@ def chambolle_pock_solver(op, x, tau, sigma, proximal_primal, proximal_dual,
 
     Notes
     -----
+    The problem of interest is:
+
+    .. math::
+
+        \\min_{x \\in X} F(K x) + G(x)
+
+    Where the technical conditions are that :math:`K` is an operator
```

Comment:
Don't start sentences with "Where", that's only a relative sentence and continues "The problem of interest is [...]".